src/usermod.c

```diff
@@ -2113,18 +2108,17 @@ static void move_mailbox (void)
 	(void) close (fd);
 
 	if (lflg) {
-		char* newmailfile;
-		size_t newsize;
+		char  *newmailfile;
 
-		newsize = strlen(prefix) + strlen(maildir) + strlen(user_newname) + 3;
-		newmailfile = XMALLOC(newsize, char);
 		if (prefix[0]) {
-			(void) snprintf (newmailfile, newsize, "%s/%s/%s",
-			                 prefix, maildir, user_newname);
-		}
-		else {
-			(void) snprintf (newmailfile, newsize, "%s/%s",
-			                 maildir, user_newname);
+			if (asprintf(&newmailfile, "%s/%s/%s",
+			             prefix, maildir, user_newname) == -1)
+			{
+				exit(EXIT_FAILURE);
+			}
+		} else {
+			if (asprintf(&newmailfile, "%s/%s", maildir, user_newname) == -1)
+				exit(EXIT_FAILURE);
```

Comment:
This isn't correctly indented

Reply:
I don't see what you mean.  Can you please paste the entire hunk from git into this conversation?  In the github interface I don't see the bad indentation.

Reply:
It seems to have disappeared. Even Github's UI states that it's outdated. Probably I was looking at some old code.